compiler/rustc_middle/src/ty/context.rs

```diff
@@ -89,62 +90,87 @@ use std::ops::{Bound, Deref};
 impl<'tcx> Interner for TyCtxt<'tcx> {
     type DefId = DefId;
     type LocalDefId = LocalDefId;
-    type AdtDef = ty::AdtDef<'tcx>;
-
     type GenericArgs = ty::GenericArgsRef<'tcx>;
+
```

Comment:
Sorry, whenever I use rust-analyzer to sort items by definition, the spacing gets all fucked up.